packages/react-mutation-mapper/src/component/revue/Revue.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { Vues } from 'genome-nexus-ts-api-client';
+import annotationStyles from '../column/annotation.module.scss';
+
+export const RevueContent: React.FunctionComponent<{ vue?: Vues }> = props => {
+    return props.vue ? (
+        <div>
+            {props.vue.comment}{' '}
+            <a
+                href={`https://pubmed.ncbi.nlm.nih.gov/${props.vue.pubmedIds[0]}/`}
+                rel="noopener noreferrer"
+                target="_blank"
+            >
+                ({props.vue.referenceText})
+            </a>
+            <ul>
+                <li>
+                    Predicted Effect: <strong>{props.vue.defaultEffect}</strong>
+                </li>
+                <li>
+                    Experimentally Validated Effect:{' '}
+                    <strong>{props.vue.variantClassification}</strong>
+                </li>
+                <li>
+                    Revised Protein Effect:{' '}
+                    <strong>{props.vue.revisedProteinEffect}</strong>
+                </li>
+            </ul>
+            <div>
+                Source:{' '}
+                <a
+                    href="https://cancerrevue.org"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                >
+                    reVUE <i className="fa fa-external-link" />
+                </a>
+            </div>
+        </div>
+    ) : (
+        <span>NA</span>
+    );
+};
+
+export function sortValue(vue: Vues | undefined): number {
+    return vue ? 1 : 0;
+}
+
+export const Revue: React.FunctionComponent<{
```

Comment:
RevueCell?

Reply:
also, seems odd that both of these props are optional.  like, maybe we should not even get to this component if fit's not Vue or if vue doesn't exist?